### PR TITLE
api information added

### PR DESCRIPTION
**Pull request by BrendonPierson**

### CODE REVIEW

File: front-end-notes/apis.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#APIs: Application Programming Interfaces
+
+##Definition 
+A way to access web-based data or software tools.
+
+An API is (usually) not a user interface. It provides software-to-software interaction, not user interactions.
+
+##REST: Representational State Transfer
+A RESTful API exposes a URL that can accept requests containing parameters (often contained in the url string).  The API will return a response based on the query parameters. The response format is often in JSON (JavaScript Object Notation).
+
```
Comment (sethdorris) [inline front-end-notes/apis.md]: Perhaps include an example call using AJAX or something?

**Comment by BrendonPierson** [comment]
There's a whole section on AJAX.

**Comment by rhondae1** [comment]
How do you link an API in your code?

**Comment by elizabrock** [comment]
Even though there is a whole other topic on AJAX, I think it would still be helpful to have a little example here to illustrate accessing an API.

**Comment by BrendonPierson** [comment]
Added an AJAX call example.

**Comment by gregkorte** [comment]
Hey Brendon. Can you please correct the spelling error in one of your titles so I can merge this doc to the curriculum?

**Pull request closed without merging**